lib/mpl_toolkits/axes_grid1/axes_divider.py

```diff
@@ -359,7 +359,7 @@ def __init__(self, fig, *args, horizontal=None, vertical=None,
         """
         Parameters
         ----------
-        fig : `matplotlib.figure.Figure`
+        fig : `~.figure.Figure`
```

Comment:
```suggestion
        fig : `~matplotlib.figure.Figure`
```

See https://matplotlib.org/stable/devel/documenting_mpl.html#referencing-types.